hexa/plugins/connector_s3/schema.py

```diff
@@ -23,7 +27,54 @@
         lastSyncedAt: DateTime
         tags: [CatalogTag!]
         icon: String!
+        Objects(
+            page: Int!,
+            perPage: Int
+        ): S3ObjectPage!
+    }
+
+    type S3Object {
+        # Base
+        id: String!
+        createdAt: DateTime!
+        updatedAt: DateTime!
+
+        # RichContent
+        owner: Organization
+        name: String!
+        shortName: String!
+        description: String!
+        countries: [Country!]
+        locale: String!
+
+        # Content
+        tags: [CatalogTag!]
+
+        # S3Object
+        bucket: S3Bucket
+        parent: S3Object
+        "Path of the object within the bucket"
+        s3Key: String
+        s3Size: Int
+        s3StorageClass: String
+        "enum: 'file' or 'directory'"
+        s3Type: String
```

Comment:
If it's an enum, let's use an enum :) https://graphql.org/learn/schema/#enumeration-types